test/api/dependency/api_test.go

```diff
@@ -0,0 +1,96 @@
+package dependency
+
+import (
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestDependencyCRUD(t *testing.T) {
+	for _, r := range Samples {
+		t.Run("Dependency_CRUD", func(t *testing.T) {
+
+			applicationFrom := api.Application{
```

Comment:
These lines actually don't create the applications. It initialize variables, but doesn't call API (so IDs will be nil). Please update to Application.Create calls, if you pass tc.ApplicationFrom/To as references (with &) it will be updated with ID "automatically".

Reply:
Missed the line! Will do that

Reply:
shall i pass in `applicationFrom/To` in Application.Create()? will that be ok?